srv/components/ha/haproxy/files/haproxy.cfg

```diff
@@ -98,8 +98,10 @@ defaults
 #---------------------------------------------------------------------
 frontend main
     # s3 server port
+{%- if pillar['cluster']['cluster_ip'] %}
```

Comment:
Also, check if `cluster_ip` is available before we set it.

Reply:
Done